package.json

```diff
@@ -21,6 +21,7 @@
         "eslint-plugin-mocha": "~8.1.0",
         "jsonschema": "~1.4.0",
         "mocha": "~8.3.2",
+        "mocha-jenkins-reporter": "^0.4.5",
```

Comment:
Do we still need `mocha-junit-reporter": "~2.0.0",` dependency?

Reply:
guess not